lib/generators/stimulus_reflex/templates/app/reflexes/%file_name%_reflex.rb.tt

```diff
@@ -66,7 +66,6 @@ class <%= class_name %>Reflex < ApplicationReflex
   <%- actions.each do |action| -%>
   def <%= action %>
   end
-  <%= "\n" unless action == actions.last -%>
```

Comment:
@julianrubisch did you remove this by accident?
![Screenshot 2023-03-13 at 15 19 35](https://user-images.githubusercontent.com/6411752/224728947-06c07027-f5f6-4711-b887-6c2c2b68f5f9.png)

Reply:
actually no, because it resulted in a dangling indentation for the last `end`.

might have to have another look at that